docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,62 @@ discretize
5f0), Point(0.41871038f0, 0.22148979f0), Point(0.4152017f0, 0.21646582f0), Point(0.4141205f0, 0.21471222f0), Point(0.41227448f0, 0.21589448f0), Point(0.40966362f0, 0.22001258f0), Point(0.40628797f0, 0.22706655f0), Point(0.40214747f0, 0.23705636f0), Point(0.40200475f0, 0.24653101f0), Point(0.40585983f0, 0.25549048f0), Point(0.41371268f0, 0.2639348f0), Point(0.4255633f0, 0.2718639f0), Point(0.4378565f0, 0.28495985f0), Point(0.4505922f0, 0.30322257f0), Point(0.46377045f0, 0.32665208f0), Point(0.47739124f0, 0.35524836f0), Point(0.5046394f0, 0.36442512f0), Point(0.5455148f0, 0.35418236f0), Point(0.60001767f0, 0.32452005f0), Point(0.66814786f0, 0.27543822f0), Point(0.7186763f0, 0.24664374f0), Point(0.75160307f0, 0.23813659f0), Point(0.76692814f0, 0.2499168f0), Point(0.7646515f0, 0.28198436f0), Point(0.7769703f0, 0.29925033f0), Point(0.8038847f0, 0.3017147f0), Point(0.84539455f0, 0.28937748f0), Point(0.9015f0, 0.26223865f0), Point(0.94408435f0, 0.24899776f0), Point(0.9731477f0, 0.24965483f0), Point(0.98869f0, 0.26420987f0), Point(0.9907113f0, 0.29266283f0), Point(0.9849871f0, 0.31338844f0), Point(0.97151726f0, 0.32638666f0), Point(0.950302f0, 0.3316575f0), Point(0.9213412f0, 0.32920095f0), Point(0.8798396f0, 0.34078467f0), Point(0.8257972f0, 0.36640862f0), Point(0.7592141f0, 0.40607283f0), Point(0.6800901f0, 0.4597773f0), Point(0.6450007f0, 0.49104902f0), Point(0.6539457f0, 0.49988794f0), Point(0.7069251f0, 0.48629412f0), Point(0.803939f0, 0.45026752f0), Point(0.877913f0, 0.4226481f0), Point(0.9288472f0, 0.40343583f0), Point(0.9567415f0, 0.39263073f0), Point(0.961596f0, 0.39023277f0), Point(0.9419039f0, 0.40523484f0), Point(0.89766514f0, 0.43763688f0), Point(0.8288798f0, 0.48743892f0), Point(0.7355478f0, 0.55464095f0), Point(0.6655121f0, 0.60063523f0), Point(0.6187727f0, 0.6254217f0), Point(0.5953296f0, 0.62900037f0), Point(0.5951828f0, 0.6113712f0), Point(0.57516366f0, 0.60261106f0), Point(0.53527224f0, 0.6027198f0), Point(0.4755085f0, 0.6116975f0), Point(0.3958725f0, 0.6295441f0), Point(0.33913234f0, 0.6398651f0), Point(0.30528808f0, 0.6426605f0), Point(0.2943397f0, 0.6379303f0), Point(0.30628717f0, 0.6256744f0), Point(0.32149008f0, 0.6093727f0), Point(0.33994842f0, 0.5890249f0), Point(0.36166218f0, 0.5646312f0), Point(0.38663134f0, 0.5361916f0), Point(0.3919681f0, 0.520893f0), Point(0.3776725f0, 0.5187355f0), Point(0.34374446f0, 0.52971905f0), Point(0.29018405f0, 0.5538437f0), Point(0.25439468f0, 0.5678829f0), Point(0.2363764f0, 0.5718367f0), Point(0.23612918f0, 0.56570506f0), Point(0.25365302f0, 0.549488f0), Point(0.2733971f0, 0.5246488f0), Point(0.29536137f0, 0.49118724f0), Point(0.3195459f0, 0.4491035f0), Point(0.34595063f0, 0.39839754f0), Point(0.3647463f0, 0.3590396f0), Point(0.37593287f0, 0.33102974f0), Point(0.37951034f0, 0.31436795f0), Point(0.37547874f0, 0.30905423f0), Point(0.36070493f0, 0.3204269f0), Point(0.33518887f0, 0.348486f0), Point(0.29893062f0, 0.3932315f0), Point(0.25193012f0, 0.45466346f0), Point(0.22926679f0, 0.47329807f0)], [])
```

Comment:
Can be made clearer by removing all those `Point` calls and `Point{2,Float32}` can be replaced with `Point2f` like above.